test/e2e/run_test.go

```diff
@@ -763,4 +763,18 @@ USER mail`
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).ToNot(Equal(0))
 	})
+
+	It("podman run --http-proxy test", func() {
+		os.Setenv("http_proxy", "1.2.3.4")
+		session = podmanTest.Podman([]string{"run", "--rm", ALPINE, "printenv", "http_proxy"})
```

Comment:
Need a `:=` here instead of `=` - it's the first instance of the variable so we need to define it

Reply:
Sorry @jamescassell my script had a bug.